client/http/api.go

```diff
@@ -80,8 +80,8 @@ const (
 	Version             = "/pd/api/v1/version"
 	operators           = "/pd/api/v1/operators"
 	safepoint           = "/pd/api/v1/gc/safepoint"
-	// Micro Service
-	microServicePrefix = "/pd/api/v2/ms"
+	// Microservice
```

Comment:
can be removed

Reply:
I'm going to keep it because it's not for exposed variables previously.